utilities/helper/accelerate.py

```diff
@@ -187,9 +188,10 @@ def daemonize_self(module, password, port, minutes, pid_file):
             pid_file.close()
             vvv("pid file written")
             sys.exit(0)
-    except OSError, e:
-        log("fork #2 failed: %d (%s)" % (e.errno, e.strerror))
-        sys.exit(1)
+    except OSError:
+        e       = get_exception()
+        message = 'fork #2 failed: %d (%s)' % (e.errno, e.strerror)
+        module.fail_json(message)
```

Comment:
By this point, I think we've already forked so we no longer want to return json on stdout.  So probably want to revert to the code that calls log() here.